fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/ElectricVehicleOperatingSystem.java

```diff
@@ -45,4 +46,11 @@ public interface ElectricVehicleOperatingSystem extends VehicleOperatingSystem {
      * Sends a request to stop charging the battery of the vehicle.
      */
     void sendChargingStopRequest();
+
+    /**
+     * Sends a request to locate all charging stations in the provided area.
+     *
+     * @param searchArea The area where the charging stations are search
+     */
+    void sendChargingStationDiscoveryRequest(GeoCircle searchArea);
```

Comment:
We should probably define the search area as a `GeoArea`.
```suggestion
    void sendChargingStationDiscoveryRequest(GeoArea searchArea);
```

Furthermore, we should also discuss what signatures are relevant.
I can think of:
- `getClosestChargingStation(GeoPosition position)` & `getClosestChargingStation()` -> returns the closest charging station either to the current position of the vehicle or to a specified position. E.g., when running low on battery.
    - Here we should also consider if we just want to consider spatial proximity or also the travel time. As charging stations at the opposite direction of a highway can actually be quite far -> `getShortestPathChargingStation(...)`
- `getChargingStationsInArea(GeoArea geoArea)` -> your use case
- `getChargingStationsWithinRange(double distance)` -> Get all charging stations that can be reached within a distance
- `getChargingStationsAlongRoute(double maxDetour)` -> Get all charging stations that are located along the route. This requires advanced routing though.